src/integration_tests/integration_tests.py

```diff
@@ -96,3 +96,8 @@ def test_post_schema(client):
 
         assert response.status_code == 200
         assert response.json() == test_schema
+
+        response = client.get(f"/v1/schema?survey_id={schema['survey_id']}")
```

Comment:
Rather than reusing the response variable, I think you should give both get responses separate, more descriptive names for readability and to avoid mutating variables more than necessary. Maybe something like `set_version_schema_response` and `latest_version_schema_response`?